Delete invalid certs during tidy operation (#665)
openbao/openbao: website/content/api-docs/secret/pki.mdx

- `tidy_cert_store` `(bool: false)` - Specifies whether to tidy up the certificate
  store.

> - `tidy_revoked_certs` `(bool: false)` - Set to true to remove all invalid revoked 
>   certs and expired certificates from storage. A revoked storage entry is considered

Reviewer: ```suggestion
- `tidy_revoked_certs` `(bool: false)` - Set to true to remove all revoked 
  and expired certificates from storage. A revoked storage entry is considered
```

I think is what I actually meant to write here :D

  invalid if the entry is empty, or the value within the entry is empty. If a
  certificate is removed due to expiry, the entry will also be removed from the
  CRL, and the CRL will be rotated.

- `tidy_invalid_certs` `(bool: false)` - Set to true to remove all invalid certificates 

>   from storage. A certificate is considered invalid if it cannot be parsed by X.509's
>   certificate parser.

Reviewer: nit:

```suggestion
  from storage. A certificate is considered invalid if it cannot be parsed by Go's
  X.509 certificate parser.
```

    "tidy_revocation_queue": false,
    "tidy_revoked_cert_issuer_associations": false,

>     "tidy_revoked_certs": false

Reviewer: ```suggestion
    "tidy_revoked_certs": false,
```
